app/lib/status_reach_finder.rb

```diff
@@ -38,7 +42,7 @@ def reached_account_ids
   end
 
   def replied_to_account_id
-    @status.in_reply_to_account_id
+    @status.in_reply_to_account_id if distributable?
   end
```

Comment:
I'm starting to wonder whether that thing is necessary at all, since replying to someone doesn't mean they will receive it, unless they are mentioned, which is always covered by `mentioned_account_ids`.

This is a remark for future consideration and not a request to change the PR, I think this change is fine.

Reply:
Actually, now that StatusReachFinder is used for distribution of Create/Update, the replied-to person *will* receive the reply, as long as it's public, and I think that only makes sense.

Reply:
Unless you mean notifications..? I think you might be right on that but it can be adjusted in a separate PR.